jupyterlab_comments/jupyterlab_comments/handlers.py

```diff
@@ -16,46 +16,63 @@
 from jupyterlab_comments.git_commands import Git
 import traitlets.config
 import json
-git = Git(config=traitlets.config.get_config()
-         )  #global instance of connection to git commands
+from pathlib import Path
+
+
+# global instance of connection to git commands
+git = Git(config=traitlets.config.get_config())
 
 
 class PreviousNamesHandler(APIHandler):
 
-  def get(self):
-    file_path = self.get_argument('file_path')
-    self.finish('Load previous names for a file (unimplemented)')
+    def get(self):
+        file_path = self.get_argument('file_path')
+        self.finish('Load previous names for a file (unimplemented)')
 
 
 class ReviewCommentsHandler(APIHandler):
 
-  def get(self):
-    file_path = self.get_argument('file_path')
-    self.finish('List review comments for a file (unimplemented)')
+    def get(self):
+        file_path = self.get_argument('file_path')
+        self.finish('List review comments for a file (unimplemented)')
 
 
 class DetachedCommentsHandler(APIHandler):
 
-  def get(self):
-    file_path = self.get_argument('file_path')
-    current_path = self.get_argument('current_path')
-    comments = git.get_comments_for_path(file_path, current_path)
-    self.finish(json.dumps(comments))
+    def get(self):
+        try:
+            file_path = self.get_argument('file_path')
+            current_path = self.get_argument('current_path')
+
+            if current_path.startswith("~"):
+                """
+                Replace the '~' with the full path to the user's home directory.
+                The modified path is needed as input to a subprocess call (setting
+                the current working directory)
+                TODO (mkalil): deal with how other OS represent home directory
+                """
+                current_path = "".join([str(Path.home()), current_path[1:]])
```

Comment:
I think this can be `current_path = os.path.expanduser(current_path)`

See details [here](https://docs.python.org/3/library/os.path.html#os.path.expanduser)